src/scheduler/mod.rs

```diff
@@ -1,4 +1,4 @@
-//! A general scheduler implementation. MMTk uses it to schedule GC-related works.
+//! A general scheduler implementation. MMTk uses it to schedule GC-related work.
```

Comment:
work packets
or 
works

Reply:
I think this is correct - 'schedule GC work'.